CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 3.1.0
+* NEW: Added TypeScript declaration file for compatibility with TypeScript projects. See: https://github.com/getsentry/raven-js/pull/610
```

Comment:
We already removed that in 3.0.0 – it's in the changelog. We just removed extra extraneous code that we missed.